src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/SwapEndpointSpec.groovy

```diff
@@ -225,118 +225,146 @@ class SwapEndpointSpec extends HealthCheckSpecification {
     @Tidy
     @Unroll
     @IterationTag(tags = [LOW_PRIORITY], iterationNameRegex = /src1/)
-    def "Able to swap #endpointsPart (#description) for two flows with the same source and different destination \
+    def "Able to swap #data.endpointsPart (src1 <-> dst2, dst1 <-> src2) for two flows with the same source and different destination \
```

Comment:
can we return the `description` variabe into the `where` block and description of the test?
if no, we need to delete `@IterationTag(tags = [LOW_PRIORITY], iterationNameRegex = /src1/)`. it is not actual in this case.